Seminar/Homework/1-8.py

```diff
@@ -0,0 +1,7 @@
+#Задача 8: Требуется определить, можно ли от шоколадки размером n × m долек отломить k долек, если разрешается сделать один разлом по прямой между дольками (то есть разломить шоколадку на два прямоугольника).
+m = int(input('Введите длину шоколадки: '))
+n = int(input('Введите ширину шоколадки: '))
+k = int(input('Введите количество долек: '))
+if k % n == 0 or k % m == 0:
+    print('Можно')
+else: print('Нельзя')
```

Comment:
Согласно PEP8, такая запись допустима, но не рекомендуема.
```suggestion
else:
    print('Нельзя')
```